setup.py

```diff
@@ -48,7 +48,7 @@
 # though that is installed out of band (managed by the extension)
 # for compatibility reasons.
 
-DEPENDENCIES = ["jsonschema==3.2.0", "packaging", "azure-iot-device~=2.5", "tqdm"]
+DEPENDENCIES = ["jsonschema==3.2.0", "packaging", "azure-iot-device~=2.5", "tqdm", "azure-storage-blob"]
```

Comment:
Do we need to take a dependency on `azure-storage-blob` ?

Reply:
I see it is being used for generating a sas uri in tests so it should be limited to `dev_requirements` rather than a general dependency addition

Reply:
Makes sense, will stick to dev_requirements